app/webapi/webapi.go

```diff
@@ -228,7 +229,7 @@ func (s *Server) checkHandler(w http.ResponseWriter, r *http.Request) {
 
 	isHtmxRequest := r.Header.Get("HX-Request") == "true"
 
-	req := spamcheck.Request{}
+	req := spamcheck.Request{CheckOnly: true}
```

Comment:
nit: this is completely unrelated to DB, shame it's not a separate PR to review.

Reply:
Yeah, this was a missing fix from an old issue. CheckOnly was added to prevent the need to clear things afterward in other places, and I found the old method here just by accident.